src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -113,6 +113,10 @@ for further details.
 	padding-right: var(--main-area-padding-right);
 	padding-top: var(--main-area-padding-top);
 	padding-bottom: 128px;
+
+	@media (--dev-dot-sidecar-up) {
+		padding-right: 16px;
```

Comment:
Great call - with the work I did as part of figuring out how to decouple parts of the layout from the breadcrumb bar height, it turned out `--main-area-padding-right` and `--main-area-padding-left` were no longer necessary to share between components (aka they were single-use CSS properties, nothing else was using them). So I've removed the CSS properties, since that felt like it cleaned things up a bit 👍